src/main/java/com/puppycrawl/tools/checkstyle/utils/ScopeUtil.java

```diff
@@ -131,23 +131,21 @@ else if (isInInterfaceOrAnnotationBlock(ast)) {
      */
     public static Scope getSurroundingScope(DetailAST node) {
         Scope returnValue = null;
-        for (DetailAST token = node.getParent();
```

Comment:
The only change in this PR right now should be here, we should hard code exactly what pitest is mutating this line to: `DetailAST token = node;`

Reply:
done !